soc/arm/nuvoton_npcx/common/soc_rom_hdr.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Header for Nuvoton NPCX EC.
+ *
+ * This header is used by Nuvoton EC Booter (ROM Code).
+ */
+
+#include <stdint.h>
+#include <kernel.h>
+
+/* Signature used by fw header */
+#define SIG_FW_EC           0x2A3B4D5E
+
+/*
+ * Error detection addresses use by header
+ * (Set them zero means do error check automatically)
```

Comment:
Is this for checking that the header is at the right address? Can you expand the comment?

Reply:
Done.